modules/rosa-adding-node-labels.adoc

```diff
@@ -7,26 +7,39 @@
 = Adding node labels
 
 
-Add or edit labels for worker nodes at any time to manage the nodes in a manner that is relevant to you. For example, you can assign types of workloads to specific nodes.
+Add or edit labels for compute nodes at any time to manage the nodes in a manner that is relevant to you. For example, you can assign types of workloads to specific nodes.
 
-Labels are assigned as key=value pairs. Each key must be unique to the object it is assigned to. Labels do not change or impact the core system values, such as a machine pool ID.
+Labels are assigned as key-value pairs. Each key must be unique to the object it is assigned to. Labels do not change or impact the core system values, such as a machine pool ID.
 
 [NOTE]
 ====
 Currently, adding node labels on an existing machine pool adds the labels to only new nodes that are created in that pool.  The existing nodes will not have the node label applied.  If you need nodes with the node label, you can either scale down the nodes in the machine pool to zero (this will not work with the 'default' machine pool), add the node label, and then scale back up to the desired amount.  You can also create a new machine pool and define the node labels at creation time.
 ====
 
+.Prerequisites
```

Comment:
I don't think the prerequisites should exist here as this doc is for OSD machinepool on UI, nothing related to rosa

Reply:
Good catch, thank you. These two prerequisites should be within the  `ifdef::openshift-rosa[]` statement that is two lines lower. I have now resolved this.